includes/class-wsuwp-shortcode-fundselector.php

```diff
@@ -48,7 +48,7 @@ public function fundselector_create_shortcode( $atts ) {
 		}
 
 		$args['unit_taxonomy'] = sanitize_key( $args['unit_taxonomy'] );
-		$args['unit_category'] = sanitize_key( $args['unit_category'] );
+		$args['unit_category'] = ! empty( $_GET['cat'] ) ? sanitize_key( $_GET['cat'] ) : null;
```

Comment:
Create a new variable instead of reusing this one. This is intended to let units set up their own priority list by setting a parameter on the shortcode (still a work in progress).